src/generate_price_reconciliation_report.py

```diff
@@ -0,0 +1,264 @@
+from configs.constants import TableNames
+import os
+import polars as pl
+import sqlite3
+from utils.utils import Utils
+
+
+def get_config_n_constant() -> tuple[str, str, str, str, str, str, str]:
+    """
+    Retrieves:
```

Comment:
Style: doctrings should have one line as a brief summary followed by a blank line.